pilot/pkg/features/pilot.go

```diff
@@ -445,7 +446,7 @@ var (
 			"These checks are both expensive and panic on failure. As a result, this should be used only for testing.",
 	).Get()
 
-	DeltaXds = env.RegisterBoolVar("ISTIO_DELTA_XDS", false,
+	DeltaXds = RegisterHiddenBoolVar("ISTIO_DELTA_XDS", false,
```

Comment:
Aren't all env vars experimental? So I am not sure why we want to hide just this one. Delta XDS does actually work (just not efficiently, but correctly).

Reply:
It is confusing to have it enable and it sends full config. I agree it is experimental but even in experimental people are expecting "delta" part works - That is the reason I was thinking about this. Otherwise if we do not want to hide, we need to expand the doc that right now "delta" part is not functional fully.

Reply:
How about we make it clear in the env var description that it does not fully work instead of registering it hidden? People should feel free to use expensive delta xds if they feel the need to

Reply:
I am fine either way. Just want to make this clear that enabling delta right now does not do a "real" delta.